src/main/java/org/fhirvp/usecase/FHIRResourceBuilderUseCase.java

```diff
@@ -0,0 +1,23 @@
+package org.fhirvp.usecase;
+
+import com.ibm.fhir.model.resource.Basic;
+import com.ibm.fhir.model.resource.ImmunizationRecommendation;
+import org.fhirvp.model.ForecastStatus;
+
+import java.util.Calendar;
+import java.util.List;
+import java.util.Optional;
+
+public interface FHIRResourceBuilderUseCase {
```

Comment:
I don't think this belongs here, it should be in the model package.

Reply:
Addressed at a9b6f4d